Patch from upstream bugzilla for
Bug 756879 - gio fam monitor: debug output can destroy functionality
https://bugzilla.gnome.org/show_bug.cgi?id=756879

aka Oracle bug 23501949:
gtk application with immodule connection dumps unnecessary strings 'II 1'

diff --git a/gio/fam/gfamfilemonitor.c b/gio/fam/gfamfilemonitor.c
index 50474c8..551b3c2 100644
--- a/gio/fam/gfamfilemonitor.c
+++ b/gio/fam/gfamfilemonitor.c
@@ -140,8 +140,6 @@ g_fam_file_monitor_is_supported (void)
 
   g_mutex_unlock (&fam_lock);
 
-  g_print ("II %d\n", fam_initialised);
-
   return fam_initialised;
 }
 
